sdk/cosmos/azure-cosmos-spark_3_2-12/docs/configuration-reference.md

```diff
@@ -27,7 +27,7 @@
 | `spark.cosmos.read.forceEventualConsistency`  | `true`    | Makes the client use Eventual consistency for read operations instead of using the default account level consistency |
 | `spark.cosmos.applicationName`      | None    | Application name  |
 | `spark.cosmos.preferredRegionsList`      | None    | Preferred regions list to be used for a multi region Cosmos DB account. This is a comma separated value (e.g., `[East US, West US]` or `East US, West US`) provided preferred regions will be used as hint. You should use a collocated spark cluster with your Cosmos DB account and pass the spark cluster region as preferred region. See list of azure regions [here](https://docs.microsoft.com/dotnet/api/microsoft.azure.documents.locationnames?view=azure-dotnet&preserve-view=true). Please note that you can also use `spark.cosmos.preferredRegions` as alias |
-| `spark.cosmos.diagnostics`      | None    | Can be used to enable more verbose diagnostics. Currently the only supported option is to set this property to `simple` - which will result in additional logs being emitted as `INFO` logs in the Driver and Executor logs.|
+| `spark.cosmos.diagnostics`      | None    | Can be used to enable more verbose diagnostics. Currently the only supported options are `simple`, `feed` and `feed_details`. The mode `simple` will result in additional logs being emitted as `INFO` logs in the Driver and Executor logs. The mode `feed` will provide per `FeedResponse` information and `feed_details` will alos add pk and id values of all items in a `FeedResponse` to the logs. Please note that any of the diagnostic modes causes the logs being emitted to increase significantly - so, there will be perf impact when enabling it.|
```

Comment:
typo: feed_details will `alos` add pk -> `also`

Reply:
Fixed in next iteration